app/components/ui/create-tale-modal/component.js

```diff
@@ -0,0 +1,158 @@
+import Ember from 'ember';
+import Component from '@ember/component';
+import {inject as service} from '@ember/service';
+import {later} from '@ember/runloop';
```

Comment:
Lint errors:
```javascript
/usr/src/node-app/app/components/ui/create-tale-modal/component.js
   12:3   error  Only string, number, symbol, boolean, null, undefined, and function are allowed as default properties  ember/avoid-leaking-state-in-ember-objects
   13:3   error  Only string, number, symbol, boolean, null, undefined, and function are allowed as default properties  ember/avoid-leaking-state-in-ember-objects
   16:21  error  Use import { computed } from '@ember/object'; instead of using Ember.computed                          ember/new-module-imports
   20:13  error  Use import { computed } from '@ember/object'; instead of using Ember.computed                          ember/new-module-imports
   58:7   error  Do not use global `$` or `jQuery`                                                                      ember/no-global-jquery
   58:7   error  '$' is not defined                                                                                     no-undef
   79:11  error  Do not use global `$` or `jQuery`                                                                      ember/no-global-jquery
   79:11  error  '$' is not defined                                                                                     no-undef
   84:19  error  Do not use global `$` or `jQuery`                                                                      ember/no-global-jquery
   84:19  error  '$' is not defined                                                                                     no-undef
  104:15  error  Do not use global `$` or `jQuery`                                                                      ember/no-global-jquery
  104:15  error  '$' is not defined                                                                                     no-undef
  150:11  error  'job' is assigned a value but never used                                                               no-unused-vars

✖ 13 problems (13 errors, 0 warnings)
```

Reply:
Thanks. The dashboard has many linter errors that can be ignored because we never configured any convention.